.github/workflows/test.yml

```diff
@@ -0,0 +1,88 @@
+name: KBase Blobstore test
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main or develop
+    branches:
+      - main
+      - master
+      - develop
+
+jobs:
+  workspace_deluxe_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - go: '1.12'
+            mongo: 'mongodb-linux-x86_64-3.6.12'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'false'
+          - go: '1.12'
+            mongo: 'mongodb-linux-x86_64-3.6.20'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'false'
+          - go: '1.12'
+            mongo: 'mongodb-linux-x86_64-3.6.20'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'true'
```

Comment:
When you update with mongo 7 you can probably get rid of the 3.6.20 wired_tiger: false matrix entry

Reply:
Will do. Also why we test 2 different v3 here?

Reply:
At the time, v12 was in production (I think? Might be v13, but pretty close anyway) and v20 was the latest 3.6 release

Reply:
👍